sql/lease.go

```diff
@@ -151,11 +152,13 @@ func (s LeaseStore) Acquire(txn *client.Txn, tableID ID, minVersion DescriptorVe
 		}
 		return nil
 	})
```

Comment:
Sure, added `roachpb.StripErrorTransaction()` and called it from `Txn.Exec()`. Could you take another look?